frontend/app/components/api/api-v3/hal-resources/work-package-resource.service.ts

```diff
@@ -268,12 +268,22 @@ export class WorkPackageResource extends HalResource {
     this.updateForm(this.$source)
       .then(form => {
         var payload = this.mergeWithForm(form);
+        const sentValues = Object.keys(this.$pristine);
```

Comment:
This seems a bit brittle to me. What if the second saving is already past this statement before the first one has entered the promise resolve function of `saveResource`. Then, the keys for the first and second change combined would be returned here. This might not cause a problem now but looks prone to cause problems later on.